### PR TITLE
Updated Using Windows topic

### PR DESCRIPTION
**Pull request by stevehoag**
Reviewed in #677

### CODE REVIEW

**Comment by stevehoag** [comment]
@mairaw Finally got this to build - okay to merge?

**Comment by mairaw** [comment]
Nope. You first need to address the merging conflicts in your content. Look at the arrows showing up throughout the content.

**Comment by stevehoag** [comment]
@mairaw  Merge conflicts addressed and build successful...

**Comment by richlander** [comment]
We need to take a closer look at your workflow. You shouldn't be getting merges as part of your work.

**Comment by mairaw** [comment]
I agree with @richlander. I just approved #699. 

BTW, we mention dnxcore50 in these docs? Is that correct?

File: docs/core/tutorials/using-on-windows.md [hunk 1]
```diff
@@ -16,31 +16,29 @@ ms.assetid: d743134a-08a3-4ff6-aab7-49f71f0568c3
...
 
-   * On the **File** menu, choose **Help**, **About Microsoft Visual Studio**.
+   * On the **Help** menu, choose **About Microsoft Visual Studio**.
 
    * In the About Microsoft Visual Studio dialog, the version number should include "Update 3" (or higher).
 
-   * If Update 3 (or higher) is not already installed, you can download [Visual Studio 2015 Update 3](https://www.visualstudio.com/news/releasenotes/vs2015-update3-vs).
-
-2. Download and install the [MSI Installer for .NET Core Tooling Preview 2](https://go.microsoft.com/fwlink/?LinkId=817245). This will install the .NET Core Tooling Preview 2 for Visual Studio 2015, which provides both .NET Core 1.0 and the Visual Studio tool set for .NET Core 1.0.
+2. Download and install the [.NET Core for Visual Studio official MSI Installer](https://go.microsoft.com/fwlink/?linkid=817245). This will install the .NET Core Tooling Preview 1 for Visual Studio 2015.
```
Comment (tdykstra) [inline docs/core/tutorials/using-on-windows.md]: Above we say preview 2, here we have preview 1

File: docs/core/tutorials/using-on-windows.md [hunk 2]
```diff
@@ -16,31 +16,29 @@ ms.assetid: d743134a-08a3-4ff6-aab7-49f71f0568c3
...
 
 ## Prerequisites
 
-* [Visual Studio 2015 Update 3](https://www.visualstudio.com/news/releasenotes/vs2015-update3-vs). If you don't have Visual Studio already, you can download [Visual Studio Community 2015](https://www.visualstudio.com/products/visual-studio-community-vs) for free.
+* Visual Studio 2015 Update 3. If you don't have Visual Studio already, you can download [Visual Studio Community 2015](https://www.visualstudio.com/downloads/download-visual-studio-vs) for free. 
 
 * NuGet Manager extension for Visual Studio. NuGet is the package manager for the Microsoft development platform including .NET Core. When you use NuGet to install a package, it copies the library files to your solution and automatically updates your project (add references, change config files, etc.).
 
 * [.NET Core Tooling Preview 2 for Visual Studio 2015](https://go.microsoft.com/fwlink/?LinkId=817245). This installs templates and other tools for Visual Studio 2015, as well as .NET Core 1.0 itself.
```
Comment (tdykstra) [inline docs/core/tutorials/using-on-windows.md]: If an item is a prerequisite, you don't need install steps in getting started, or vice versa.

File: docs/core/tutorials/using-on-windows.md [hunk 3]
```diff
@@ -16,31 +16,29 @@ ms.assetid: d743134a-08a3-4ff6-aab7-49f71f0568c3
...
 
 ## Prerequisites
 
-* [Visual Studio 2015 Update 3](https://www.visualstudio.com/news/releasenotes/vs2015-update3-vs). If you don't have Visual Studio already, you can download [Visual Studio Community 2015](https://www.visualstudio.com/products/visual-studio-community-vs) for free.
+* Visual Studio 2015 Update 3. If you don't have Visual Studio already, you can download [Visual Studio Community 2015](https://www.visualstudio.com/downloads/download-visual-studio-vs) for free. 
 
 * NuGet Manager extension for Visual Studio. NuGet is the package manager for the Microsoft development platform including .NET Core. When you use NuGet to install a package, it copies the library files to your solution and automatically updates your project (add references, change config files, etc.).
```
Comment (tdykstra) [inline docs/core/tutorials/using-on-windows.md]: NuGet is here as both an install step and a prerequisite

File: docs/core/tutorials/using-on-windows.md [hunk 4]
```diff
@@ -92,9 +90,9 @@ A solution using only .NET Core projects
 
 7. Open the context menu for the **References** node and choose **Manage NuGet Packages**.
 
-8. Choose "nuget.org" as the **Package source**, and choose the **Browse** tab. Check the **Including prerelease** checkbox, and then browse for **xUnit** version 2.1.0, and then click **Install**. 
+8. Choose "nuget.org" as the **Package source**, and choose the **Browse** tab. Check the **Including prerelease** checkbox, and then browse for **xUnit** version 2.2.0 or newer, and then click **Install**.
```
Comment (tdykstra) [inline docs/core/tutorials/using-on-windows.md]: The UI is "Include", not "Including"

**Comment by tdykstra** [comment]
Comments on lines that don't show up in the diff:
1. In Writing the Test Project -- on other tutorial comments, we've been advised to avoid splat strings ("1.0.0.-*") even for project references
2. Also in Writing the Test Project, why do we restore packages immediately before adding one?  
3. In "A mixed .NET Core..." section -- extra "the" -- open the the context menu
4. In multiple places we manually click Restore Packages -- is that necessary?  Seems to happen automatically.
5. In multiple places "Including" needs to change to "Include" regarding the NuGet UI
6. Typo "contaxt menu"
7. For each scenario ("Moving a PCL to a Netstandard", "Moving a library from netstandard 1.4 to 1.3", etc.) where the tutorial jumps into how-to steps, it might be helpful to start by explaining the scenario and why you might want to do that particular operation.

File: docs/core/tutorials/using-on-windows.md (hunk 1 above)
Comment (stevehoag) [inline docs/core/tutorials/using-on-windows.md]: Fixed

File: docs/core/tutorials/using-on-windows.md (hunk 2 above)
Comment (stevehoag) [inline docs/core/tutorials/using-on-windows.md]: By design - readers may want to review the prereqs before deciding to install anything

File: docs/core/tutorials/using-on-windows.md (hunk 3 above)
Comment (stevehoag) [inline docs/core/tutorials/using-on-windows.md]: Same as previous.

File: docs/core/tutorials/using-on-windows.md (hunk 2 above)
Comment (tdykstra) [inline docs/core/tutorials/using-on-windows.md]: If you have the install link in the prereqs section, you're covered both ways.

File: docs/core/tutorials/using-on-windows.md (hunk 4 above)
Comment (stevehoag) [inline docs/core/tutorials/using-on-windows.md]: Fixed in 4 places

File: docs/core/tutorials/using-on-windows.md (hunk 2 above)
Comment (stevehoag) [inline docs/core/tutorials/using-on-windows.md]: Makes sense - I'll add the links.

**Comment by stevehoag** [comment]
@tdykstra fixed 3,4,5,6 above.

@bleroy @cartermp  Can you comment on Tom's issue 1,2 and 7 above (since you wrote the original article).

**Comment by mairaw** [comment]
I also had a question about the use of dnxcore in the article. Isn't that deprecated now?

**Comment by tdykstra** [comment]
In the section "A mixed ..." it has
    replace "frameworks": { "netstandard1.5" with "frameworks": { "netstandard1.4".
which assumes the project was created with 1.5, but when I create a project, the template creates it as 1.6

**Comment by richlander** [comment]
This PR has a ton of merge artifacts in it. I don't know what happens upon squash, but we're not going to merge a PR like that.

I'm going to try to fix it.

**Comment by richlander** [comment]
Check out #733.

We can you use that PR or you can reset this branch and take and re-push those changes for this PR.

**Comment by stevehoag** [comment]
@richlander Thanks for fixing this. Let's use #733 and close this one.

**Pull request closed without merging**